app/views/search/_view_all_link.html.erb

```diff
@@ -2,9 +2,14 @@
   <div class="results-more">
   <% if @results['total'] > 0 %>
     <% return unless @results['total'] > @per_box %>
+    <% if Flipflop.enabled?(:local_browse) && @results['local_view_more'] %>
```

Comment:
Yeah. Good idea. 🚮

I had to take the logic out of the model because of the cache, but now that you called me out on it I have some ideas of how to clean up the views while still not causing cacheing issues. I think I'll handle that refactor when I work on the related features / bug fixes.